locales/lv/messages.json

```diff
@@ -153,6 +153,7 @@
     },
     "backupFileUnmigratable": {
         "message": "Rezerves kopijas failu uzģenerēja iepriekšējās versijas konfigurators un tas nav migrējams. Piedodiet."
+        "description": "Edited in crowdin by EGParadox."
```

Comment:
This needs a comma at the end of the previous line or it will break the JSON format.